### PR TITLE
Make sure temp dir is definable since the amount of data can grow big.

### PR DESCRIPTION
**Pull request by smetj**
I have seen the tempfiles created here grow pretty huge, beyond the capacity of /tmp so it's probably wise to make the destination definable.
It uses the "tmp_path" value from thruk.conf

### CODE REVIEW

File: lib/Thruk/Backend/Provider/Mysql.pm [hunk 1]
```diff
@@ -400,7 +400,7 @@ sub get_logs {
     # logfiles into tmp file
     my($fh, $filename);
     if($options{'file'}) {
-        ($fh, $filename) = tempfile();
+        my($fh, $filename) = tempfile(DIR => ($c->config->{'tmp_path'} || '/tmp'));
```
Comment (sni) [inline lib/Thruk/Backend/Provider/Mysql.pm]: please remove the "my" here. Thats done in line 401 already.

**Comment by smetj** [comment]
removed 2nd my

**Comment by sni** [comment]
Seems like its not that easy. There is no $c context available in that get_logs sub.

**Comment by smetj** [comment]
Yeah indeed, I'm running a patched version which has path hard coded.  Now how it should be but I overlooked the c$ context.

Looking through the code there's quite a bit that needs to be changed in this case.
I'll close this pull request and see what I can do on my side.

**Comment by sni** [comment]
I am also a bit afraid this might break existing setups. In OMD for example, tmp_dir points to a tmpfs in memory, so writing the tmp logfiles there might be a bad idea.

**Pull request closed without merging**